### DISCUSSION TITLE
Separate Victory Music Fanfare

### DISCUSSION THREAD

**Discussion by Ganbare-Lucifer** [ikemen-engine/Ikemen-GO#928] [Feature Request]
As the title implies, make an option to play the stage victory music as a fanfare and play the system victory music afterwards in the victory screen.

**Comment by jankespro12** [comment]
Kinda like in Old Mortal Kombat games?

**Comment by Ganbare-Lucifer** [comment]
Not only there, but in Street Fighter 2 and Killer Instinct as well, not sure how many of these games had that feature.

**Comment by jankespro12** [comment]
also Capcom vs. SNK arcade version has that too
